why-promptlayer/evaluation-page.mdx

```diff
@@ -3,7 +3,7 @@ title: "Evaluations"
 icon: "question"
 ---
 
-The Evaluations page allows users to run a prompt against a batch of test cases from a JSON or CSV file. This documentation will guide you through the various features and options available on the Evaluations page.
+The Evaluations page allows users to run a prompt against a Dataset, a batch of test cases built from a JSON or CSV file. This documentation will guide you through the various features and options available on the Evaluations page.
```

Comment:
Let's rephrase the first sentence

"The Evaluations page lets users execute a prompt against a dataset, which is built from test cases in either a JSON or CSV file."